priv/working_groups/eef.md

```diff
@@ -0,0 +1,5 @@
+name: Erlang Ecosystem Foundation
```

Comment:
I'm not sure what this is for. My guess would be a default post for the index perhaps. Could you explain?

Reply:
Yes, it is the default topic for news section.

Reply:
Ahh, I see. Great! Let's change the email to `eef@erlef.org`?